utam-core/src/main/java/utam/core/declarative/translator/ErrorMessageProvider.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root
+ * or https://opensource.org/licenses/MIT
+ */
+package utam.core.declarative.translator;
+
+/**
+ * API responsible for building compiler error messages
+ *
+ * @author elizaveta.ivanova
+ * @since 238
+ */
+public interface ErrorMessageProvider {
```

Comment:
Are there other interfaces we anticipate will be `ErrorMessageProvider`s? If not, why break out the separate interface?

Reply:
removed